gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/workflow/ExecuteGobblinWorkflow.java

```diff
@@ -37,5 +38,5 @@
 public interface ExecuteGobblinWorkflow {
   /** @return the number of {@link WorkUnit}s discovered and successfully processed */
   @WorkflowMethod
-  int execute(Properties props, EventSubmitterContext eventSubmitterContext);
+  ExecGobblinStats execute(Properties props, EventSubmitterContext eventSubmitterContext);
```

Comment:
given the workflow is named `ExecuteGobblin`, the `ExecGobblinStats` actually does sound reasonable.